packages/sdk/src/browser-wallets/okx/index.ts

```diff
@@ -0,0 +1,152 @@
+import { Psbt } from "bitcoinjs-lib";
+
+import { getAddressFormat } from "../../addresses";
+import { BrowserWalletNetwork } from "../../config/types";
+import {
+  BrowserWalletExtractTxFromNonFinalizedPsbtError,
+  BrowserWalletNotInstalledError,
+  OrditSDKError,
+} from "../../errors";
+import { BrowserWalletSignResponse, WalletAddress } from "../types";
+import { OKXSignPSBTOptions } from "./types";
+
+function isInstalled(): boolean {
+  if (typeof window === "undefined") {
+    throw new OrditSDKError("Cannot call this function outside a browser");
+  }
+  return typeof window.okxwallet !== "undefined";
+}
+
+async function getOKXWalletProvider(
```

Comment:
Async can be removed for this function as there are no promises used

Reply:
Oh yes. Changed!